libbeat/docs/outputconfig.asciidoc

```diff
@@ -684,6 +684,12 @@ The configurable ClientID used for logging, debugging, and auditing purposes. Th
 
 The number of concurrent load-balanced Kafka output workers.
 
+===== codec
+
+Output codec configuration. If `codec` section is missing, events will be json encoded.
```

Comment:
I would say, "If the `codec` section is missing..." [global change to all instances of this description]